### PR TITLE
Removal of Cleric: Disciple of Life Handling

### PR DESCRIPTION
**Pull request by Aeristoka**
DnDBeyond now natively supports Disciple of Life

### CODE REVIEW

**Comment by kakaroto** [comment]
Good bye friend, we had a good run, you shall be missed.

**Comment by jkomg** [comment]
It's honestly too bad - they're just lumping Disciple of Life into the main healing of the spell instead of showing it as separate in the same roll the way Beyond20 does - which is much more elegant a solution. 

This still doesn't show Blessed Healer and makes you have to think about it, too, because you can't just take the Beyond20 showing of Disciple of Life and add that to yourself, you gotta go "Okay, 2 + spell level to me..." which is clearly a small problem or nonissue but...

Sometimes the "fix" seems lazy compared to the workaround.

TL;DR: we appreciate you @kakaroto

**Comment by Aeristoka** [comment]
> It's honestly too bad - they're just lumping Disciple of Life into the main healing of the spell instead of showing it as separate in the same roll the way Beyond20 does - which is much more elegant a solution.
> 
> This still doesn't show Blessed Healer and makes you have to think about it, too, because you can't just take the Beyond20 showing of Disciple of Life and add that to yourself, you gotta go "Okay, 2 + spell level to me..." which is clearly a small problem or nonissue but...
> 
> Sometimes the "fix" seems lazy compared to the workaround.
> 
> TL;DR: we appreciate you @kakaroto

Yeah, when I was verifying my code stripping for this I had that same feeling, "this doesn't show anywhere near the info as we are in Beyond20...", sucks.

**Comment by jkomg** [comment]
and I'm sorry I didn't give you credit for this code, @Aeristoka!

**Comment by Aeristoka** [comment]
> and I'm sorry I didn't give you credit for this code, @Aeristoka!

No worries 😄 , I didn't write the original code, just cleaned it out when it became redundant 😭 .

**Comment by kakaroto** [comment]
yeah, I know! I had the same feelings about it as well. Ideally, someone should poke ddb to say "put it as a separate healing like you can have a spell with multiple damages".

**Comment by kakaroto** [comment]
It's definitely an improvement for users who aren't beyond20 users, but I also bet some people might still think they need to add it manually and not realize it's already included.
